hack/Dockerfile

```diff
@@ -143,10 +136,15 @@ RUN set -ex \
 	&& mkdir -p $internalDirs \
 	&& chown -R archivematica $internalDirs
 
+RUN set ex  \
+    && npm install -g npm@9.8.1
```

Comment:
Why 9.8.1 and not the latest 9.x, e.g. 9.9.2?

Reply:
Thank you and noted all your suggestions.  I made changes accordingly in https://github.com/artefactual/archivematica/pull/1875